jobrunner/config/common.py

```diff
@@ -10,7 +10,18 @@
 BACKENDS = os.environ.get("BACKENDS", "test,tpp,emis").strip().split(",")
 
 # Used for tracing in both agent and controller
-VERSION = os.environ.get("VERSION", "")
+# This refers to a file created in the docker image by Dockerfile
+JOBRUNNER_VERSION_FILE_PATH = os.environ.get(
+    "JOBRUNNER_VERSION_FILE_PATH", "/app/metadata/version.txt"
+)
```

Comment:
It would be slightly neater to do this as:
```py
JOBRUNNER_VERSION_FILE_PATH = Path(
    os.environ.get(
        "JOBRUNNER_VERSION_FILE_PATH", "/app/metadata/version.txt"
    )
)
```
Then you can call `.exists()` on it instead of using `os.path.exists` and you don't need to wrap it again below.

Reply:
nice, just added